static/app/views/performance/traceDetails/content.tsx

```diff
@@ -94,14 +99,65 @@ class TraceDetailsContent extends React.Component<Props, State> {
   }
 
   renderTraceNotFound() {
-    const {meta} = this.props;
+    const {meta, traceEventView, traceSlug, organization, location} = this.props;
 
     const transactions = meta?.transactions ?? 0;
     const errors = meta?.errors ?? 0;
 
     if (transactions === 0 && errors > 0) {
+      const eventView = {...traceEventView};
+      eventView.query = `trace:${traceSlug} !event.type:transaction `;
+
       return (
-        <LoadingError message={t('The trace you are looking contains only errors.')} />
+        <React.Fragment>
+          <Alert type="error" icon={<IconInfo size="md" />}>
+            <ErrorLabel>
+              {t('The trace cannot be shown when all events are errors.')}
+            </ErrorLabel>
+            <DiscoverQuery
+              eventView={traceEventView}
+              orgSlug={organization.slug}
+              location={location}
+              referrer="api.trace-view.errors"
```

Comment:
You'll also want to add this to the list of allowed referrers [here](https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_events.py#L15) or it'll be coerced to the default referrer. The reason we do this is so we don't have arbitrary referrers and limit its cardinality.

Reply:
Updated to use [api.trace-view.errors-view](https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_events.py#L30), since it isn't being used anywhere else in the codebase 🙂